database/migrations/2025_03_07_163259_create_city_airline_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('city_airline', function (Blueprint $table) {
```

Comment:
```suggestion
        Schema::create('airline_city', function (Blueprint $table) {
```
I think these have to be in alphabetical order for Laravel to correctly assume relationships